src/main/java/subway/config/SubwayConfig.java

```diff
@@ -0,0 +1,28 @@
+package subway.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+import subway.auth.ui.LoginInterceptor;
+import subway.path.domain.Path;
+
+@Configuration
+public class SubwayConfig implements WebMvcConfigurer {
+
+    @Bean
+    public LoginInterceptor loginInterceptor(){
+        return new LoginInterceptor();
+    }
+
+    @Override
+    public void addInterceptors(InterceptorRegistry registry) {
+        registry.addInterceptor(loginInterceptor()).addPathPatterns("/favorites/**");
+        registry.addInterceptor(loginInterceptor()).addPathPatterns("/members/me");
```

Comment:
아래와 같이 다수의 pattern 을 설정하면 됩니다.
```suggestion
        registry.addInterceptor(loginInterceptor()).addPathPatterns("/favorites/**", "members/me");
```